src/glassflow/pipelines.py

```diff
@@ -28,6 +30,9 @@ def __init__(self, glassflow_client, pipeline_id: str,
         self.pipeline_id = pipeline_id
         self.organization_id = self.glassflow_client.organization_id
         self.pipeline_access_token = pipeline_access_token
+        self._consume_retry_delay_minimum = 1
+        self._consume_retry_delay_current = 1
+        self._consume_retry_delay_max = 60
```

Comment:
It is not critical but it is nice to describe the numbers: are they in seconds, minutes